src/uct/tcp/tcp_iface.c

```diff
@@ -283,6 +275,7 @@ static UCS_CLASS_DEFINE_NEW_FUNC(uct_tcp_iface_t, uct_iface_t, uct_md_h,
                                  uct_worker_h, const uct_iface_params_t*,
                                  const uct_iface_config_t*);
 
+
```

Comment:
Is there a  need for extra line here? Single line is used in the rest of the fille.